packages/tests/html-imports/html/imports/es-module-1.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 import {message} from './es-module-2.js';
```

Comment:
```suggestion
import './es-module-2.js';
```

I don't *think* this was relevant to the test.